packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -36,7 +36,7 @@ function logToConsole(data: LogData) {
 export const DEFAULT_LOGGER_CONFIG: LoggerConfigInternal = {
   logFunction: logToConsole,
   level: LogLevel.warn,
-  strict: false,
+  strict: true,
```

Comment:
Better to set to `true` by default and add information how to disable it with link to docs - will be added with respective docs pages in the #6467 PR.